modules/nodes-scheduler-node-selectors-pod.adoc

```diff
@@ -181,6 +182,7 @@ $ oc get nodes -l type=user-node,region=east
 NAME                          STATUS   ROLES    AGE   VERSION
 ip-10-0-142-25.ec2.internal   Ready    worker   17m   v1.26.0
 ----
+endif::openshift-rosa,openshift-dedicated[]
```

Comment:
This becomes a single-step (with substeps) procedure in the ROSA/OSD builds. Maybe there's a way to similarly make these individual `.` (non-sub) steps too, as in the other example.